src/Surfnet/StepupMiddleware/ApiBundle/Identity/Entity/RaCandidate.php

```diff
@@ -94,6 +79,10 @@ class RaCandidate implements JsonSerializable
      */
     public $email;
 
+    private function __construct()
+    {
+    }
```

Comment:
Was adding a private constructor important? And why?

Reply:
This is to prevent instantiating an Entity which shouldn't be initiated at all because a view is used.